diff --git a/azurelinuxagent/common/utils/cryptutil.py b/azurelinuxagent/common/utils/cryptutil.py
index 0471a1f..7be0228 100644
--- a/azurelinuxagent/common/utils/cryptutil.py
+++ b/azurelinuxagent/common/utils/cryptutil.py
@@ -83,7 +83,7 @@ class CryptUtil(object):
         elif not os.path.exists(trans_prv_file):
             raise IOError(errno.ENOENT, "File not found", trans_prv_file)
         else:
-            cmd = ("{0} cms -decrypt -in {1} -inkey {2} -recip {3} "
+            cmd = ("{0} cms -decrypt -in {1} -inkey {2} "
                    "| {4} pkcs12 -nodes -password pass: -out {5}"
                    "").format(self.openssl_cmd, p7m_file, trans_prv_file,
                               trans_cert_file, self.openssl_cmd, pem_file)
