diff --git a/lib/conversion.c b/lib/conversion.c
index ce0f5b3..753b2e0 100644
--- a/lib/conversion.c
+++ b/lib/conversion.c
@@ -134,7 +134,7 @@ static bool tpm2_convert_pubkey_ssl(TPMT_PUBLIC *public, pubkey_format format, c
         goto error;
     }
 
-#if OPENSSL_VERSION_NUMBER < 0x1010000fL /* OpenSSL 1.1.0 */
+#if OPENSSL_VERSION_NUMBER < 0x1010000fL || defined(LIBRESSL_VERSION_NUMBER) /* OpenSSL 1.1.0 */
     ssl_rsa_key->e = e;
     ssl_rsa_key->n = n;
 #else
diff --git a/lib/tpm_kdfa.c b/lib/tpm_kdfa.c
index 3d1ed50..3f4d0af 100644
--- a/lib/tpm_kdfa.c
+++ b/lib/tpm_kdfa.c
@@ -54,7 +54,7 @@ static const EVP_MD *tpm_algorithm_to_openssl_digest(TPMI_ALG_HASH algorithm) {
 static HMAC_CTX *hmac_alloc()
 {
     HMAC_CTX *ctx;
-#if OPENSSL_VERSION_NUMBER < 0x1010000fL /* OpenSSL 1.1.0 */
+#if OPENSSL_VERSION_NUMBER < 0x1010000fL || defined(LIBRESSL_VERSION_NUMBER) /* OpenSSL 1.1.0 */
     ctx = malloc(sizeof(*ctx));
 #else
     ctx = HMAC_CTX_new();
@@ -62,7 +62,7 @@ static HMAC_CTX *hmac_alloc()
     if (!ctx)
         return NULL;
 
-#if OPENSSL_VERSION_NUMBER < 0x1010000fL
+#if OPENSSL_VERSION_NUMBER < 0x1010000fL || defined(LIBRESSL_VERSION_NUMBER)
     HMAC_CTX_init(ctx);
 #endif
 
@@ -71,7 +71,7 @@ static HMAC_CTX *hmac_alloc()
 
 static void hmac_del(HMAC_CTX *ctx)
 {
-#if OPENSSL_VERSION_NUMBER < 0x1010000fL
+#if OPENSSL_VERSION_NUMBER < 0x1010000fL || defined(LIBRESSL_VERSION_NUMBER)
     HMAC_CTX_cleanup(ctx);
     free(ctx);
 #else
