diff -urN rdf-spec-0.3.5.orig/lib/rdf/spec/writer.rb rdf-spec-0.3.5/lib/rdf/spec/writer.rb
--- rdf-spec-0.3.5.orig/lib/rdf/spec/writer.rb	2012-02-25 20:24:26.192825047 +0900
+++ rdf-spec-0.3.5/lib/rdf/spec/writer.rb	2012-02-25 20:56:32.505515544 +0900
@@ -1,5 +1,6 @@
 require 'rdf/spec'
 require 'fileutils'
+require 'tmpdir'
 
 share_as :RDF_Writer do
   include RDF::Spec::Matchers
@@ -29,13 +30,13 @@
   describe ".open" do
     before(:each) do
       RDF::Util::File.stub!(:open_file).and_yield(StringIO.new("foo"))
-      @dir = File.join(File.expand_path(File.dirname(__FILE__)), "tmp")
+      @dir = File.join(Dir.tmpdir, "rdf-spec")
       Dir.mkdir(@dir)
       @basename = File.join(@dir, "foo")
     end
     
     after(:each) do
-      FileUtils.rm_rf(File.join(File.expand_path(File.dirname(__FILE__)), "tmp"))
+      FileUtils.rm_rf(@dir)
     end
 
     it "yields writer given file_name" do
